src/registrar/admin.py

```diff
@@ -187,43 +200,48 @@ class MyUserAdmin(BaseUserAdmin):
         "email",
         "Permissions",
         "is_active",
-        "is_staff",
-        "is_superuser",
+        "groups",
         "Important dates",
         "last_login",
         "date_joined",
     ]
 
     def get_list_display(self, request):
-        if not request.user.is_superuser:
-            # Customize the list display for staff users
-            return (
-                "email",
-                "first_name",
-                "last_name",
-                "is_staff",
-                "is_superuser",
-                "status",
-            )
-
-        # Use the default list display for non-staff users
-        return super().get_list_display(request)
+        # The full_access_permission perm will load onto the full_access_group
+        # which is equivalent to superuser. The other group we use to manage
+        # perms is cisa_analysts_group. cisa_analysts_group will never contain
+        # full_access_permission
+        if request.user.has_perm("registrar.full_access_permission"):
```

Comment:
It seems a little weird that Django chose to manage these as strings. It might be a bit of a refactor but we should really be storing these in a constant or a literal

Reply:
Ideally, yes. Not worth it at this point imo.

Reply:
Fair enough